packages/auth/src/core/strategies/action_code_settings.test.ts

```diff
@@ -60,14 +60,27 @@ describe('core/strategies/action_code_settings', () => {
       _setActionCodeSettingsOnRequest(auth, request, {
         handleCodeInApp: true,
         iOS: {
-          bundleId: 'my-´bundle'
+          bundleId: 'my-bundle'
```

Comment:
preferably to create a magic constant for string values that repeated multiple times.
e.g. const TEST_BUNDLE_ID = 'my-bundle';

Reply:
Done. Thanks Liubin!